Network Activity/DNS Activity/Zeek/dns_log/zeek_dns.log.event.parsed.json

```diff
@@ -0,0 +1,45 @@
+{
+    "time": 1591367999.305988,
+    "actor.user.uuid": "CMdzit1AMNsmfAIiQc",
+    "src_endpoint.ip": "192.168.4.76",
+    "src_endpoint.port": 36844,
+    "dst_endpoint.ip": "192.168.4.1",
+    "dst_endpoint.port": 53,
+    "connection_info.protocol_name": "udp",
+    "answers.packet_uid": 19671,
+    "unmapped.rtt": 0.06685185432434082,
+    "query.hostname": "testmyids.com",
+    "query.class": "C_INTERNET",
+    "query.type": "A",
+    "rcode": "NOERROR",
+    "answers.rdata": [
+        "31.3.245.133"
+    ],
+    "answers.ttl": [
+        3600
+    ],
+    "activity_id": false,
+    "answers.flag_ids.1": false,
```

Comment:
Considering answers is an array of objects, it would be something like - 

```
[
    {
        "flag_ids": [
            3,
            4
        ],
        "packet_uid": 19671,
        "rdata": "31.3.245.133",
        "ttl": "3600"
    }
]
```

}
}

Reply:
updated answers field to reflect your example above